charts/camunda-platform/charts/optimize/templates/deployment.yaml

```diff
@@ -160,3 +160,16 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
 {{- end }}
+      initContainers:
```

Comment:
Please add a flag to enable/disable it.

Reply:
And let's have it disabled by default and enable it with 8.3.

Reply:
done